tests/core/data_layer/test_data_rpc.py

```diff
@@ -308,6 +309,9 @@ async def test_keys_values_ancestors(one_wallet_node_and_rpc: nodes_with_port, b
         assert dic["0x" + key3.hex()] == "0x" + value3.hex()
         assert dic["0x" + key4.hex()] == "0x" + value4.hex()
         assert dic["0x" + key5.hex()] == "0x" + value5.hex()
+        assert len(keys["keys"]) == len(dic)
+        for key in keys["keys"]:
+            assert "0x" + key.hex() in dic
```

Comment:
Related to above, shouldn't `key` be JSON serializable and not bytes here?  maybe...